docs/architecture/adr-013-msg-authorization.md

```diff
@@ -0,0 +1,220 @@
+# ADR 13: Msg Authorization
+
+## Changelog
+
+- 2019-11-06: Initial ADR
+
+## Context
```

Comment:
The context should state the (concise) motivation for why we believe this is needed.